### PR TITLE
pull request for Diamond aligner PKGBUILD

### PR DESCRIPTION
**Pull request by kbipinkumar**
## Involved packages

 - diamond-aligner

## Involved issue

 #86 

## Details

- [x] Tested in the local machine (largest 16G RAM) and it is passed without any issue
- [x] Provide New Package
- [x] Fix the Packages
  - [x] PKGBUILD
  - [x] lilac.yaml
  - [x] lilac.py
- [ ] Would like to continue to work with us

### CODE REVIEW

File: BioArchLinux/diamond-aligner/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+source=("$pkgname-$pkgver.tar.gz::https://github.com/bbuchfink/diamond/archive/v$pkgver.tar.gz")
+sha256sums=('cc8e1f3fd357d286cf6585b21321bd25af69aae16ae1a8f605ea603c1886ffa4')
+
+prepare() {
+  # shellcheck disable=2154
+  mkdir -p "$srcdir"/diamond-$pkgver/build
+}
+
+build() {
+  cd "$srcdir"/diamond-$pkgver/build || exit 1
```
Comment (sukanka) [inline BioArchLinux/diamond-aligner/PKGBUILD:23]: I suggest removing codes like `|| exit 1` for simplicity.

**Comment by sukanka** [comment]
* I recommend renaming the executable as `diamond-aligner` since there is another package called `diamond` in AUR and this package itself is named `diamond-aligner`.
* You have to add a `lilac.yaml` and a `lilac.py`. You can have a look at [bamtools](https://github.com/BioArchLinux/Packages/tree/master/BioArchLinux/bamtools)

**Comment by starsareintherose** [comment]
> 

I don't suggest use `diamond-aligner`, we don't need to rename the software, it even doesn't conflict with other software in core or extra, we shouldn't rename it.

**Comment by starsareintherose** [comment]
You can see https://aur.archlinux.org/packages/diamond is orphan, few people use it, just take it over.

**Comment by starsareintherose** [comment]
add a lilac.yaml only is okay, see here
```
build_prefix: extra-x86_64
maintainers:
  - github: your_github
    email: your_non_outlook_email
pre_build_script: |
  update_pkgver_and_pkgrel(_G.newver.lstrip('v'))
  run_cmd(['updpkgsums'])
post_build_script: |
  git_add_files('PKGBUILD')
  git_commit()
update_on:
  - source: github
    github: bbuchfink/diamond
    use_max_tag: true
    prefix: 'v' 
```

**Comment by starsareintherose** [comment]
I have done almost all, you just need to change the folder

**Comment by starsareintherose** [comment]
Don't worry about AUR, I have take it over using BioArchLinuxBot, just push commit is okay. For previous diamond, I now create one https://aur.archlinux.org/packages/python-diamond

**Pull request merged**